app/assets/stylesheets/components/_button.scss

```diff
@@ -18,3 +18,23 @@
   color: grey;
   text-decoration: none;
 }
+
+.btn-custom-green {
+  color: white;
+  border-radius: 12px;
+  top: 9px;
+  position: relative;
+  padding: 12px 36px;
+  font-size: $font_medium;
+  font-family: $displaylight;
+  background: #00CCBC !important;
```

Comment:
Could you put the color to a variable please?

Reply:
done